src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+import InfoBadge16 from '../../icon/fill/InfoBadge16';
+// @ts-ignore flow import
+import Toggle from '../../components/toggle';
+import Tooltip from '../../components/tooltip';
+
+import messages from './messages';
+
+import './AdvancedContentInsightsToggle.scss';
+
+interface Props {
+    hasTooltip?: boolean;
+    isAdvancedContentInsightsChecked?: boolean;
```

Comment:
this prop could be `isChecked` since `AdvancedContentInsights` is part of the component name so it's not needed to include it in the prop name

note: the change would only be for this component. it still makes sense to keep it in UnifiedShareForm

Reply:
thanks! one more place sorry:
https://github.com/box/box-ui-elements/blob/bf909deaf9d8bd7428b99327634dddd0b83a8834/src/features/unified-share-modal/UnifiedShareForm.js#L762
`isChecked={isAdvancedContentInsightsChecked}`

Reply:
@tjuanitas my bad! Updated